src/main/java/net/dv8tion/jda/core/managers/impl/AudioManagerImpl.java

```diff
@@ -65,8 +66,8 @@
 
     public AudioManagerImpl(Guild guild)
     {
-        this.guild = guild;
-        this.api = (JDAImpl) guild.getJDA();
+        this.guild = (GuildImpl) guild;
```

Comment:
The purpose of the interfaces is that user can't see the methods and fields used by their implementations as they have to be public because they have to be accessible from other parts of the API. You shouldn't and also can't create your own implementations of these classes (well, technically you can but as JDA will never use them it doesn't make any sense).